tools/msys2_fetch_and_build_all_tx16s.sh

```diff
@@ -1,12 +1,45 @@
 #! /usr/bin/env bash
 
-## Bash script to show how to get EdgeTX source from GitHub (main branch),
+## Bash script to show how to get EdgeTX source from GitHub,
 ## how to build TX16S firmware, Companion, Simulator, radio simulator
 ## library and how to create an installation package.
 ## Let it run as normal user in MSYS2 MinGW 64-bit console (blue icon).
+## this script is working on any branch from 2.8 (icluded)
```

Comment:
Try "This script works properly only for branches stemming from EdgeTX v2.8 or newer."